zerver/webhooks/slack_incoming/view.py

```diff
@@ -41,15 +41,15 @@ def api_slack_incoming_webhook(
         try:
             val = request.body.decode(request.encoding or "utf-8")
         except UnicodeDecodeError:  # nocoverage
-            raise JsonableError(_("Malformed payload"))
+            return slack_error_response(JsonableError(_("Malformed payload")))
```

Comment:
I think you want to instead have an outer try/except turning JsonableEerror into `slack_error_response`, since the `.tame()` scales scattered through also should include the `ok` payload.